source/guides/guides/command-line.md

```diff
@@ -554,13 +554,14 @@ cypress verify
 
 ## `cypress version`
 
-Output both the versions of the installed Cypress binary application and the npm module.
-In most cases they will be the same, but they could be different if you have installed a different version of the npm package and for some reason could not install the matching binary.
+Prints the installed Cypress binary application version, the Cypress NPM module version, Electron version used to build Cypress, and the bundled Node version. In most cases the binary and the NPM package versions will be the same, but they could be different if you have installed a different version of the npm package and for some reason could not install the matching binary.
```

Comment:
Mention package instead of npm specifically.

```suggestion
Prints the installed Cypress binary version, the Cypress package version, the version of Electron used to build Cypress, and the bundled Node version. 

In most cases the binary and the package versions will be the same, but they could be different if you have installed a different version of the package and for some reason failed to install the matching binary version.
```